src/renderer/components/Transaction/TransactionModal.vue

```diff
@@ -142,8 +144,8 @@ export default {
         if (this.isSuccessfulResponse(response)) {
           this.storeTransaction(this.transaction)
 
-          if (data && data.accept.length === 0 && data.broadcast.length > 0) {
-            this.$warn(messages.warningBroadcast)
+          if (data && (data.accept.length === 0 || data.broadcast.length === 0)) {
```

Comment:
If the transaction is not accepted, but broadcasted, it would be better to display a different message.

Reply:
Well, it shows a warning message `this.$warn(messages.warningLowFee, 10000)`:
<img width="653" alt="50374217-6b7c2280-05f3-11e9-9d03-929d7ef4b991" src="https://user-images.githubusercontent.com/3462869/50817750-85c13a80-132d-11e9-872b-91cc57bb6451.png">

If you want to show one message when only accepted and a different one when only broadcasted, imo this will be too technical for the end user and it's not necessary. For instance, if we say `Tx was broadcasted but not accepted` will be technically wrong, because the current peer didn't accept it but some other could. Also, 99% of the users have no clue what `broadcasted` means.

Reply:
Yes, but if the transaction is not accepted by the peer, but broadcasted and we show the same message than when it's broadcasted, it could happen that none of the peers accept it, so to be correct, we should display a different message.

Reply:
Yes, but if the transaction is not accepted by the peer, but broadcasted and we show the same message than when it's broadcasted, it could happen that none of the peers accept it, so to be correct, we should display a different message.

Reply:
OK, please decide with the others exactly what messages you want to show and I'll add them.

Reply:
Unless @alexbarnsley or @luciorubeens have a different opinion, I'd go with the shorter version.